source/retrieve-data.txt

```diff
@@ -0,0 +1,196 @@
+.. _java-rs-retrieve:
+
+=============
+Retrieve Data
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: code examples, read, search, cursor
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-long+} to retrieve
+data from a MongoDB collection by using read operations. You can call the
+``find()`` method to retrieve documents that match a set of criteria.
```

Comment:
S: I would leave out this second sentence; it feels tacked on and doesn't add anything critical